DigitalLearningSolutions.Data/DataServices/RegistrationDataService.cs

```diff
@@ -21,9 +21,10 @@ public class RegistrationDataService : IRegistrationDataService
         private readonly IDbConnection connection;
         private readonly IUserDataService userDataService;
 
-        public RegistrationDataService(IDbConnection connection)
+        public RegistrationDataService(IDbConnection connection, IUserDataService userDataService)
         {
             this.connection = connection;
+            this.userDataService = userDataService;
         }
 
         public string RegisterNewUserAndDelegateAccount(DelegateRegistrationModel delegateRegistrationModel)

```

Comment:
There is a TODO for this ticket two lines below this comment. The only calling locations in non-test code that has not been addressed in previous tickets is `CreateDelegateAccountForAdmin` in `RegistrationService`. I think changes to this method should probably be done in [HEEDLS-900](https://softwiretech.atlassian.net/browse/HEEDLS-900) rather than here. Should I do it here or update the TODO?